tests/test_astroid.py

```diff
@@ -12,38 +13,27 @@ class TestAstroid(test_mark_tokens.TestMarkTokens):
   is_astroid_test = True
   module = astroid
 
+  nodes_classes = astroid.ALL_NODE_CLASSES
```

Comment:
Why is `astroid.NodeNG` not sufficient? (if it really is not, would you mind adding a comment?)

Reply:
`astroid.NodeNG` doesn't resolve, it has to be imported from `astroid.node_classes`, and I just find it less readable anyway (what does NodeNG mean?)